docs/docs/firefly-iii/pages-and-features/rules.md

```diff
@@ -19,9 +17,12 @@ A rule must spring into action at the right time! This is decided by triggers th
 * When the amount is more than *X*.
 * When the budget is *X*.
 
-Rules can be set to be "strict" or not. If a rule is set to be strict, ALL triggers must match for the rule to fire. If a rule is not strict, ANY trigger is enough.
+All triggers are case-insensitive.
+
+Rules can be set to be "strict" or not. If a rule is set to be strict, ALL triggers must match for the rule to fire. If a rule is not strict, ANY trigger is enough. 
 
-All triggers are case-insensitive
+!!! info
+    This can be useful if you wish to match ANY description, for example. If your transactions are described "groceries" or "went shopping" or "got food" then a *non-strict* rule could match all of them by adding 3 triggers with different content.
```

Comment:
```suggestion
    This can be useful if you wish to match multiple descriptions with the same rule, for example. If your transactions are described "groceries" or "went shopping" or "got food" then a *non-strict* rule could match all of them by adding 3 triggers with different content.
    As another example, make multiple triggers for different grocery store expense accounts, and as action put them all in the same budget.
```